src/Digdir.Domain.Dialogporten.Infrastructure/Persistence/Development/HostingEnvironmentExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Reflection;
+using Microsoft.Extensions.Hosting;
+
+namespace Digdir.Domain.Dialogporten.Infrastructure.Persistence.Development;
+
+internal static class HostingEnvironmentExtensions
+{
+    public static bool ShouldRunDevelopmentHostedService(this IHostEnvironment environment)
+    {
+        // Only run in development environments, but not when using Janitor
+        return environment.IsDevelopment() && !(Assembly.GetEntryAssembly()?.GetName().Name ?? "").Contains("Janitor");
```

Comment:
Er ikke helt fan av at infrastruktur "kjenner til" presentasjonslag. Dette er jo kun et DX hensyn, som gjør det "ok". Et annet forslag er å bruke LocalDeveloperSettings og appsettings.development.json i Janitor prosjektet for å skru av spesifikk funksjonalitet. Mer i tråd med de linjene vi alt har. Lar det være opp til deg @elsand

Reply:
Er et hack ja, og siden jeg i etterkant av dette introduserte settings for å disable kjøring av hosted services, så gir det mer mening å bare bruke det her også. Det er egentlig bare de to synk-jobbene som er relevant å deaktivere; f.eks. databasemigrering vil en nok uansett gjøre. Så skal fikse :)